backend/db.js

```diff
@@ -0,0 +1,26 @@
+const { Pool } = require('pg');
+require('dotenv').config(); // Load environment variables
+
+// Database connection string
+const connectionString = process.env.DATABASE_URL || 'postgresql://SaudiRailwaysSystem_owner:qRA8uemT2aMg@ep-wild-frog-a5w1q21s.us-east-2.aws.neon.tech/SaudiRailwaysSystem?sslmode=require';
```

Comment:
We can store the DATABASE_URL in a .env file like this

```
DATABASE_URL=postgresql://<username>:<password>@<host>/<database>?sslmode=require
```

And if you do that your connectionString becomes:

```javascript
const connectionString = process.env.DATABASE_URL;
```